### PR TITLE
[PORT] Security badges

### PR DESCRIPTION
**Pull request by Marshmellow105**
## About The Pull Request

Gives the HOS, Warden, Officers and Detectives new security badges which you can wave around when you shout about the NTSD. The sprites come from eris, I just added a red stripe to the security badge.

Here is the PR I pulled the sprites and code from: https://github.com/discordia-space/CEV-Eris/pull/8411

image

## Why It's Good For The Game

A little roleplay flavour for the security department.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

Item sprites (Detective, Security)
image

Worn sprites (Same order)
image

In game (Detectives badge)
image
image

In game (Security badge)
image
image

In game whilst worn
image

</details>

## Changelog
:cl: marshmellow10, thevandie
add: Added badges for the Detective and Security
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
you should specify the port source PR

**Comment by Marshmellow105** [comment]
done

**Comment by Geatish** [comment]
Shouldnt there be a special badge for HoS too you think? Also, since we are starting to rack up suitables, maybe we should port that multiple suit items from tg

**Comment by Marshmellow105** [comment]
> Shouldnt there be a special badge for HoS too you think? Also, since we are starting to rack up suitables, maybe we should port that multiple suit items from tg

I could probably throw gold on the officer badge for the HOS

as for the multiple suit items, that apparently requires rightclick/combat mode which is the equivalent of a level 300 raid boss to my level 1 coder class

**Comment by Geatish** [comment]
> that apparently requires rightclick/combat mode which is the equivalent of a level 300 raid boss to my level 1 coder class

That sounds way out of my range, which i will be stupid enough to try it

**Comment by Marshmellow105** [comment]
the HOS badge has been added by request

image

**Comment by Marshmellow105** [comment]
Think of the fun you can have.

badge

**Comment by ClownMoff** [comment]
Make a clown one

**Comment by Geatish** [comment]
a final request, make them fit inside of a wallet, if you haven't already

**Comment by Marshmellow105** [comment]
> a final request, make them fit inside of a wallet, if you haven't already and add a weight class to them....

Do you have an example of an item that isnt an ID that can go into a wallet so I can look it up when I get off of work?



> Make a clown one

I couldn't do it justice. Imagine a bananium badge with a little afro like the clown PDA.

**Comment by Geatish** [comment]
> Do you have an example of an item that isnt an ID that can go into a wallet so I can look it up when I get off of work?

Just add `/obj/item/clothing/accessory/badge/officer` to the list of things in wallet.dm
like this
image

**Comment by Marshmellow105** [comment]
my man 🤝

File: code/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -409,3 +409,27 @@
...
+/obj/item/clothing/accessory/badge/officer
+	name = "\improper Security badge"
+	desc = "A badge of the NanoTrasen Security Division, made of silver and set on false black leather."
+	icon_state = "officerbadge"
+	worn_icon_state = "officerbadge"
+
+/obj/item/clothing/accessory/badge/officer/attack_self(mob/user)
+    if(Adjacent(user))
+        user.visible_message("<span class='notice'>[user] shows you \the: [icon2html(src, viewers(user))] [src.name].</span>", "<span class='notice'>You show \the [src.name].</span>")
+    add_fingerprint(user)
```
Comment (PowerfulBacon) [inline code/modules/clothing/under/accessories.dm]: Call ..() instead of manually adding the fingerprint

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Review by EvilDragonfiend — APPROVED** [review]

**Pull request merged**